orttraining/orttraining/core/optimizer/graph_transformer_config.h

```diff
@@ -0,0 +1,28 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/optimizer/graph_transformer_config.h"
+
+namespace onnxruntime {
+
+namespace training {
+
+struct TrainingGraphTransformerConfiguration : public GraphTransformerConfiguration {
```

Comment:
Do we need a base class?

Reply:
As mentioned above the base class is only to help not create a dependence in onnxruntime namespace. Alternatively, we could move the PropagateCastOps to orttraining namespace. I added the derived class because there are some options that are only relevant to training.